config/defaults.mk

```diff
@@ -505,7 +511,7 @@ CUDA_LIB = -lcusparse
 
 # HIP library configuration
 HIP_OPT =
-HIP_LIB = -L$(HIP_DIR)/lib $(XLINKER)-rpath,$(HIP_DIR)/lib -lhipsparse
+HIP_LIB = -L$(HIP_DIR)/lib $(XLINKER)-rpath,$(HIP_DIR)/lib -lhipsparse -lhipblas
```

Comment:
For CUDA, we should probably add `-lcublas` to `CUDA_LIB`.

Reply:
Done in bc729465d6117a4b9a1bf5235ee4239f3d021c1c.